modules/imaging_uploader/test/TestPlan.md

```diff
@@ -42,42 +75,44 @@
+      the Number Of MincCreated and Number Of MincInserted columns, then check that the Number Of MincCreated column has a 
       clickable link that takes the user to the MRI violations page, displaying the violated scans of this upload.
     [Manual Testing]      
-12. Go to the Candidate Profile page and search for the CandID of the candidate to which the scan belongs. Make sure 
+13. Go to the Candidate Profile page and search for the CandID of the candidate to which the scan belongs. Make sure 
     that column scan done is set to yes. Click on the 'Yes' link and verify that it takes you to the Imaging Browser
     page with all the valid scans for this candidate loaded in the result table.
     [Manual Testing]
-13. Go in the profile details for the candidate found in step 10 and go into the visit table. Check that the MR Scan 
+14. Go in the profile details for the candidate found in step 13 and go into the visit table. Check that the MR Scan 
     done column is set to 'Yes' for the visit at which the scan was done. Click on the Yes link and make sure you 
     are taken to the Imaging Browser page with the valid scans performed for that candidate at that visit loaded 
     in the result table.
     [Manual Testing]
-14. Go the the Imaging Uploader page and search for the upload done in step 10 using (in turn) the CandID, PSCID and
-    visit label. Verify that the upload done in step 10 is shown in the table each time. 
+15. Go the the Imaging Uploader page and search for the upload done in step 11 using (in turn) the CandID, PSCID and
+    visit label. Verify that the upload done in step 11 is shown in the table each time. 
     [Manual Testing]
-15. Repeat steps 10-14 but with a .tgz and .zip file.
+16. Repeat steps 11-15 but with a `.tgz` and `.zip` file.
     [Manual Testing]
-16. In the Admin/Configuration module, set the 'ImagingUploader Auto launch' option to 'Yes'.
-    Repeat steps 10-15 to verify that the insertion scripts work with the Auto Launch option.
+17. In the Admin/Configuration module, set the 'ImagingUploader Auto launch' option to 'Yes'.
+    Repeat steps 11-16 to verify that the insertion scripts work with the Auto Launch option.
     [Manual Testing]
-17. Upload a valid, anonymized .tar.gz DICOM archive but with a CandID that does not match the one in the archive 
-    (PSCID and visit label should be correct though). Check that an appropriate message is written in the Console 
-    Output.
+18. Upload a valid, anonymized `.tar.gz` DICOM archive but with a patient name (PSCID, CandID, Visit label) that does not
+    match the one stored in the DICOM files of the archive (for example, patient name DICOM header = `MTL0001_111111_V01`
+    but `.tar.gz` name being `MTL0002_222222_V01` which corresponds to another valid candidate).
+    Check that an appropriate message is written in the Console Output (i.e. "`The PatientName read from the DICOM header
```

Comment:
@cmadjar @victoriafoing 
this is the wrong message; Imaging Uploader does not read the DICOM headers at all.
It only checks the fields in the form against what is in the database and against the filename.

Reply:
In this case, the autolaunch is on so once the scan is uploaded, the imaging pipeline is launched and then in the console window, you get a message saying that PatientName in the DICOMs do not start by the provided patient name in the archive name and the pscid, candid, visit label fields.

Reply:
@MounaSafiHarab what should I change it to?

Reply:
@victoriafoing 
as @cmadjar pointed out, this step is a subset of the step before, meaning auto launch is on, so the insertion scripts launch the pipeline.
Your message is therefore appropriate, I would just reword "Console Log" to 'Log Viewer" as this specific message appears in the Log Viewer part. But this is very minor.